app/views/quotes/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<% @quotes.each do |quote| %>
+  <div>
```

Comment:
Similar to @Dkendal's note here: https://github.com/StemboltHQ/quoteboard/pull/1/files#r48580873, instead of a `<div>` you could use a `<blockquote>`.